Monitor failing in integration

### Description

```
Caused by: io.grpc.StatusRuntimeException: UNKNOWN
        at io.grpc.stub.ClientCalls.toStatusRuntimeException(ClientCalls.java:268)
        at io.grpc.stub.ClientCalls.getUnchecked(ClientCalls.java:249)
        at io.grpc.stub.ClientCalls.blockingUnaryCall(ClientCalls.java:135)
        at com.hedera.hashgraph.sdk.Executable.lambda$new$0(Executable.java:107)
        at com.hedera.hashgraph.sdk.Executable.execute(Executable.java:422)
        ... 14 common frames omitted
Caused by: java.lang.UnsupportedOperationException: null
        at io.grpc.netty.AbstractHttp2Headers.isEmpty(AbstractHttp2Headers.java:40)
        at io.netty.handler.codec.http2.DefaultHttp2ConnectionDecoder$FrameReadListener.onHeadersRead(DefaultHttp2ConnectionDecoder.java:419)
        at io.netty.handler.codec.http2.DefaultHttp2ConnectionDecoder$FrameReadListener.onHeadersRead(DefaultHttp2ConnectionDecoder.java:352)
        at io.netty.handler.codec.http2.Http2InboundFrameLogger$1.onHeadersRead(Http2InboundFrameLogger.java:56)
        at io.netty.handler.codec.http2.DefaultHttp2FrameReader$2.processFragment(DefaultHttp2FrameReader.java:476)
        at io.netty.handler.codec.http2.DefaultHttp2FrameReader.readHeadersFrame(DefaultHttp2FrameReader.java:484)
        at io.netty.handler.codec.http2.DefaultHttp2FrameReader.processPayloadState(DefaultHttp2FrameReader.java:253)
        at io.netty.handler.codec.http2.DefaultHttp2FrameReader.readFrame(DefaultHttp2FrameReader.java:159)
        at io.netty.handler.codec.http2.Http2InboundFrameLogger.readFrame(Http2InboundFrameLogger.java:41)
        at io.netty.handler.codec.http2.DefaultHttp2ConnectionDecoder.decodeFrame(DefaultHttp2ConnectionDecoder.java:188)
```

### Steps to reproduce

`./gradlew :monitor:run`

### Additional context

_No response_

### Hedera network

other

### Version

main

### Operating system

None

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -29,6 +29,7 @@ plugins {
 
 // Can't use typed variable syntax due to Dependabot limitations
 extra.apply {
+    set("grpcVersion", "1.59.1")
     set("gson.version", "2.8.9") // Temporary until Apache jclouds supports gson 2.9
     set("logback.version", "1.4.14") // Temporary until next Spring Boot version
     set("mapStructVersion", "1.5.5.Final")
@@ -42,6 +43,7 @@ extra.apply {
 // Creates a platform/BOM with specific versions so subprojects don't need to specify a version when using a dependency
 dependencies {
     constraints {
+        val grpcVersion: String by rootProject.extra
         val mapStructVersion: String by rootProject.extra
         val protobufVersion: String by rootProject.extra
         val reactorGrpcVersion: String by rootProject.extra
@@ -70,7 +72,7 @@ dependencies {
         api("commons-io:commons-io:2.15.1")
         api("io.cucumber:cucumber-bom:7.14.1")
         api("io.github.mweirauch:micrometer-jvm-extras:0.2.2")
-        api("io.grpc:grpc-bom:1.59.1")
+        api("io.grpc:grpc-bom:$grpcVersion")
         api("io.hypersistence:hypersistence-utils-hibernate-62:3.6.1")
         api("io.projectreactor:reactor-core-micrometer:1.1.0")
         api("io.swagger:swagger-annotations:1.6.12")

--- a/buildSrc/src/main/kotlin/java-conventions.gradle.kts
+++ b/buildSrc/src/main/kotlin/java-conventions.gradle.kts
@@ -46,7 +46,8 @@ repositories {
 
 dependencyManagement {
     imports {
-        mavenBom("io.grpc:grpc-bom:1.59.0")
+        val grpcVersion: String by rootProject.extra
+        mavenBom("io.grpc:grpc-bom:${grpcVersion}")
         mavenBom(SpringBootPlugin.BOM_COORDINATES)
     }
 }

